maintainers/maintainer-list.nix

```diff
@@ -13001,4 +13001,10 @@
     githubId = 88109321;
   };
 
+  ursi =
+    { name = "Mason Mackaman";
+      email = "masondeanm@aol.com";
+      github = "ursi";
+      githubId = 17836748;
+    };
```

Comment:
Different commit: `maintainers: add ursi`

Reply:
Please add yourself as a maintainer by adding an alphabetically sorted entry to `maintainers/maintainer-list.nix` and commit this change into a separate commit with the message ``maintainers: add NAME`` into this PR.

Reply:
my bad, looking from the bottom, I didn't realize it was roughly alphabetical.